# feature_selection_gwas
Genome-wide association Studies (GWAS) is a research approach aimed at comparing the genomes of different individuals to identify genetic markers indicative of phenotypes or disease risks. The use of GWAS simulations provides a means to assess the efficacy of different GWAS methods under various scenarios. Here, I present computer simulated GWAS data, including both noise and SNPs (simulated Single Nucleotide Polymorphisms). Notably, the dataset has an equal number of columns, encompassing both SNPs and noise data.
In the field of computer analysis and statistical computation, the presence of numerous columns or features that lack relevance can significantly hinder the interpretation of results. The inclusion of non-informative information complicates statistical computations, making the process time-consuming. As humans, it becomes challenging to discern which columns (features) are pivotal and which are extraneous in simulated data.
To overcome this challenge, I conducted experiments applying various computer methods to gauge their performance and extract meaningful information. The results indicate that each algorithm employs a distinct approach, yielding different features. Consequently, the conclusion drawn is that the diversity in methods results in the identification of unique, relevant features, enhancing our ability to extract valuable insights from simulated GWAS data. 
